### ISSUE TITLE
How can we write visibleIf condition using not (!) operator

### ISSUE THREAD

**Issue by bappajubcse05** [surveyjs/surveyjs#1076]
### Are you requesting a feature, reporting a bug or ask a question?
Question

### What is the current behavior?
Trying to write below condition for field6 -

`{field1} = 'select' and ({field2} != 'value1' or !(['x', 'y', 'z'].contains({field3}) && (['a', 'b', 'c'].contains({field4}) or ['m', 'n', 'o'].contains({field5}))))`

Scenarios:
1) when field2 is 'value1' --> filed6 should not be visible.
2) when field3 in ('x', 'y', 'z') and (field4 in ['a', 'b', 'c']. or field5 in ['m', 'n', 'o']) --> field6 should not be visible

But it seems !(not) operator is not working properly.

https://surveyjs.io/Service/EditSurvey/835ada28-2782-4881-9561-e4d5b15d403f

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
your_code_here

```

### Specify your

* browser:
* browser version:
* surveyjs platform (angular or react or jquery or knockout or vue):
* surveyjs version:

**Comment by andrewtelnov** [comment]
@bappajubcse05 Unfortunately we do not support "!" and "not" yet :(
You have to reverse the expression in the brackets.
Additionally, you will have to use contains and notcontains as:
`{question1} contains 'a' or {question1} contains 'b'`
or
`{question1} notcontains 'a' and {question1} notcontains 'b'`

I mark it as a suggestion.

PS: `['x', 'y', 'z'].contains({field3}` should be `['x', 'y', 'z'] contains {field3}`
 `['x', 'y', 'z'].notcontains({field3}` should be `['x', 'y', 'z'] notcontains {field3}`

Thank you,
Andrew

**Comment by bappajubcse05** [comment]
@andrewtelnov - thanks.
Could you please suggest me an alternative way to get the same result.

**Comment by andrewtelnov** [comment]
@bappajubcse05 Sure:
```
{field1} = 'select' and ({field2} != 'value1' or ((['x', 'y', 'z'] notcontains {field3} or ['a', 'b', 'c'] notcontains {field4}) and ['m', 'n', 'o'] notcontains {field5})
```
Something like that.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@bappajubcse05 Should be available in v1.0.23

Thank you,
Andrew
